src/host/exe/exemain.cpp

```diff
@@ -163,6 +202,13 @@ int CALLBACK wWinMain(
 
     ConsoleCheckDebug();
 
+    // Set up OutOfProc COM server stuff in case we become one.
+    // WRL Module gets going right before winmain is called, so if we don't
+    // set this up appropriately... other things using WRL that aren't us
+    // could get messed up by the singleton module and cause unexpected errors.
+    _comServerExitEvent.create();
+    auto& module = OutOfProcModuleWithRegistrationFlag<REGCLS_SINGLEUSE>::Create(&_releaseNotifier);

```

Comment:
1. It's not dangerous to do this for every conhost on every single launch, correct?
2. Should this only run in out-of-box builds
3. should this be behind the conint "do def app" check?

Reply:
1. This particular section just sets up the metadata and factories. 
2. We could scope it if you want. It won't happen inbox unless someone manually launches `conhost.exe -Embeddding`. But I could make it so that can't happen.
3. If I don't do 2, I could do 3. Thoughts?

Reply:
Going to block embedding and acceptance on inbox INSIDE WINDOWS def and put it behind the check.